plugins/jobs/src/js/components/JobsOverviewError.tsx

```diff
@@ -0,0 +1,15 @@
+import * as React from "react";
+
+import RequestErrorMsg from "#SRC/js/components/RequestErrorMsg";
+
+import JobsPage from "./JobsPage";
+
+export default class JobsOverviewLoading extends React.Component<{}> {
+  render() {
+    return (
+      <JobsPage>
+        <RequestErrorMsg />
```

Comment:
Would love to see a proper message here. It will say `Can't connect with the server` all the time

Reply:
I agree, but we had the same discussion on @Fabs catalog refactoring… IMHO we should create a new task for this.